ERCS/erc-XXXXscriptregistry.md

```diff
@@ -0,0 +1,150 @@
+---
```

Comment:
```suggestion
---
eip: 7738
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.